guide/resources/wallet-interoperability.md

```diff
@@ -0,0 +1,196 @@
+---
+layout: guide
+title: Wallet interoperability
+description: Writing tips for authors of the Bitcoin Design Guide.
+nav_order: 10
+parent: Resources
+permalink: /guide/resources/wallet-interoperability/
+main_classes: -no-top-padding
+image: /assets/images/guide/resources/wallet-interoperability-preview.jpg
+---
+
+<!--
+
+Editor's notes
+
+An overview of important interaction points between different Bitcoin and other applications.
+
+Designers should think beyond the software they are working on and strongly consider these
+areas, as it is extremely likely that users interact with multiple applications over time.
+
+Illustration sources
+
+https://www.figma.com/file/qzvCvqhSRx3Jq8aywaSjlr/Bitcoin-Design-Guide-Illustrations-CO?node-id=1024%3A6795
+
+-->
+
+{% include picture.html
+   image = "/assets/images/guide/wallet-interoperability/wallet-interoperability.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/wallet-interoperability@2x.jpg"
+   mobile = "/assets/images/guide/wallet-interoperability/wallet-interoperability-mobile.jpg"
+   mobileRetina = "/assets/images/guide/wallet-interoperability/wallet-interoperability-mobile@2x.jpg"
+   alt-text = "Several devices exchanging data"
+   width = 1600
+   height = 700
+   layout = "full-width"
+%}
+
+# Wallet interoperability
+
+Diverse applications with different philosophies, feature sets and approaches are fantastic for Bitcoin. It allows the many different users around the world to choose the applications that best fit their needs. For this to be practical, a minimum of interoperability should be worked towards, for smoother interactions with fewer mistakes.
+
+Here are some common scenarios:
+- Using different wallets for desktop and mobile
+- Paying someone who uses different software
+- Migrating to a different device, operating system or application
+- Multi-signature wallets that require multiple devices and applications to exchange data
+- Using your own node
+
+Each of these requires that applications are designed and developed to be open, ideally relying on standardized formats.
+
+So here are some ways to allow users to more seamlessly navigate different software configurations.
+
+## Wallet import and export
+
+<div class="center" markdown="1">
+
+{% include image.html
+   image = "/assets/images/guide/wallet-interoperability/key-import-export.jpg"
+   retina = "/assets/images/guide/wallet-interoperability/key-import-export@2x.jpg"
+   alt-text = "Transfer of wallet data between applications"
+   width = 400
+   height = 300
+   layout = "float-right-desktop"
+%}
+
+Allow for wallets generated in one application to be easily restored in another application. Over the years, Bitcoin applications have implemented various technical details in different ways, partly because standards take time to evolve. See [Wallets Recovery](https://walletsrecovery.org) as an illustration of the problem.
```

Comment:
This is extremely important section. Should we maybe expand it a bit and add that following BIP's that are widely adopted helps solve this? I am so glad you included that link to walletsrecovery.org it's a great resource, but we may want to emphasize some of their points in this section?

I mean we provided a problem here but not a concrete solution is what I'm thinking.

Reply:
Good point. I added a bit more info, recommending that wallets should put extra effort to empower users with all the info they need to recover wallets in the future. Big topic. Daniel had mentioned he is thinking of creating a dedicated page for wallet recovery, this would be a good spot to link there.

Reply:
Excellent idea, I think a dedicated wallet recovery page is needed.

Reply:
@danielnordh two votes here for your idea for a page about wallet recovery.